rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -1742,9 +1743,10 @@ int LuaSyncedCtrl::DestroyUnit(lua_State* L)
 /***
  * @function Spring.TransferUnit
  * @number unitID
- * @number newTeamID
+ * @number newTeamId
  * @bool[opt=true] given if false, the unit is captured.
- * @treturn nil
+ * @bool[opt=false] adjustUnitLimit if true, sets newTeam.maxUnits++ and oldTeam.maxUnits-- before transfer
```

Comment:
Nitpick: "if true, increases the new team unit limit by 1 and respectively decreases the old team limit by 1"

Reply:
or just "if true, also transfer the limit slot"

Reply:
Sounds good, I think the ++ and -- usage is something we should avoid in documentation